fattmerchant-ios-sdk/Cardpresent/Mock/MockDriver.swift

```diff
@@ -12,12 +12,32 @@ class MockDriver: MobileReaderDriver {
 
   weak var mobileReaderConnectionStatusDelegate: MobileReaderConnectionStatusDelegate?
 
+  var currentlyConnectedReader: MobileReader?
+
   var reader: MobileReader? = MobileReader(name: "Reader",
-                            firmwareVersion: "FakeFirmwareVersion",
-                            make: "FakeMake",
-                            model: "FakeModel",
+                            firmwareVersion: "FakeFirmwareVersion1",
+                            make: "FakeMakeR1",
+                            model: "FakeModelR1",
                             serialNumber: "FakeSerialNumber")
 
+  var reader2: MobileReader? = MobileReader(name: "Reader 2",
+                                           firmwareVersion: "fwv2",
+                                           make: "FakeMakeR2",
+                                           model: "FakeModelR2",
+                                           serialNumber: "TJCJX23UWLE9Y69J")
+
+  var reader3: MobileReader? = MobileReader(name: "Reader 3",
+                                           firmwareVersion: "fwv3",
+                                           make: "FakeMakeR3",
+                                           model: "FakeModelR3",
+                                           serialNumber: "GDES4RB9K4QPGWWW")
+
+  var reader4: MobileReader? = MobileReader(name: "Reader 4",
+                                           firmwareVersion: "fwv4",
+                                           make: "FakeMakeR4",
+                                           model: "FakeModelR4",
+                                           serialNumber: "7ZP3Y2JM5G227U9E")
+
```

Comment:
Instead of hardcoding the readers, could we create a function that generates one? Hardcoding this to four would mean that the user has to be aware that the limit is four. If we were to increase the limit, we'd also need to hardcode more readers